src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml.cs

```diff
@@ -93,5 +106,229 @@ internal override void Close()
                 this.ViewModel.Dispose();
             }
         }
+
+        internal void Generate_Editorconfig(object sender, System.Windows.RoutedEventArgs e)
+        {
+            var editorconfig = new StringBuilder();
+            var optionSet = this.ViewModel.ApplyChangedOptions(this.OptionService.GetOptions());
+
+            GenerateEditorconfig_CoreSettings(optionSet, _language, editorconfig);
+            GenerateEditorconfig_DotNetSettings(optionSet, _language, editorconfig);
+
+            // Generate language specific settings
+            _getCurrentEditorConfigOptions(optionSet, editorconfig);
```

Comment:
Nit: Probably rename to `_getLangaugeSpecificEditorConfigOptions` so you don't need the above comment.